packages/core/saved-objects/core-saved-objects-migration-server-internal/src/zdt/model/model.test.mocks.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const StageMocks = {
+  init: jest.fn().mockImplementation((state: unknown) => state),
```

Comment:
NIT: We will be forced to manually add every stage to this mock when we'll add new ones, but I couldn't find a way to avoid it.

Reply:
That's a good idea, didn't think of it 😄 . Adapted